sitback-furniture/src/components/button/button.module.scss

```diff
@@ -0,0 +1,83 @@
+.btn{
+    border: none;
+    font-weight: 600;
+    font-size: 1rem;
+    padding: 0.7rem 1.5rem;
+    cursor: pointer;
+    border-bottom-right-radius: 0.5rem;
+}
+
+.btn-wrapper .btn.small{
+    padding: 0.5rem 1rem;
+    font-size: 0.7rem;
+    font-weight: 500;
+    border-bottom-right-radius: 0.3rem;
+}
+
+.header,.side-menu{
+
+    .btn{
+        padding: 0 0 1rem 0;
+    }
+    &:hover{
+        .tab{
+            // background-color: #fff08b;
+            background-color: #484848;
```

Comment:
Could you move all the color codes to a separate variables?

Reply:
moved all the color codes to variable